app-modules/ai/tests/Feature/Filament/Pages/AIAssistantPageTest.php

```diff
@@ -1214,3 +1215,73 @@
 
     Bus::assertNotDispatched(PrepareAiThreadEmailing::class);
 });
+
+it('can not email a thread to a super admin', function () use ($setUp) {
+    ['thread' => $thread] = $setUp();
+
+    $superAdmin = User::factory()->create();
+
+    $superAdmin->assignRole('authorization.super_admin');
+
+    Livewire::test(PersonalAssistant::class)
+        ->callAction('emailThread', [
+            'targetType' => AiThreadShareTarget::User->value,
+            'targetIds' => [$superAdmin->getKey()],
+        ], arguments: [
+            'thread' => $thread->getKey(),
+        ])
+        ->assertHasActionErrors(['targetIds'])
+        ->assertSeeText('Super admin users cannot have a thread emailed to them.');
+});
+
+it('can not clone a thread to a super admin', function () use ($setUp) {
+    ['thread' => $thread] = $setUp();
+
+    $superAdmin = User::factory()->create();
+
+    $superAdmin->assignRole('authorization.super_admin');
+
+    Livewire::test(PersonalAssistant::class)
+        ->callAction('cloneThread', [
+            'targetType' => AiThreadShareTarget::User->value,
+            'targetIds' => [$superAdmin->getKey()],
+        ], arguments: [
+            'thread' => $thread->getKey(),
+        ])
+        ->assertHasActionErrors(['targetIds'])
+        ->assertSeeText('Super admin users cannot have a thread emailed to them.');
+});
+
+it('Super admin users do not show up in email user search', function () use ($setUp) {
+    ['thread' => $thread] = $setUp();
```

Comment:
If you don't use `$thread` just call `$setUp()` without destructuring.

Reply:
Done.